lib/index.js

```diff
@@ -10,12 +10,15 @@ const outputChanges = require('./outputChanges')
 const outputResult = require('./outputResult')
 
 module.exports = async ({
+  config = false,
```

Comment:
You can assign a better default here:

```suggestion
  config = 'jest.config.js',
```

Reply:
resolved in db585672e17064f8ff5c01d3451fa2ba48e131fd